plugins/hotkeyshortcuts.koplugin/_meta.lua

```diff
@@ -0,0 +1,6 @@
+local _ = require("gettext")
+return {
+    name = "hotkeyshortcuts",
+    fullname = _("Hotkey Shortcuts"),
```

Comment:
Style:
```suggestion
    fullname = _("Hotkey shortcuts"),
```

But why not just hotkey?
```suggestion
    fullname = _("Hotkeys"),
```

Reply:
I think it was originally just `Shortcuts` but you guys said it could be confused with other existing shortcuts, so I just added the word hotkey (which might be less known among non-english speakers (according to me, that is) ) but kept the shortcuts so it was "clearer". I don't ming just `Hotkeys` though